public/src/client/topic.js

```diff
@@ -71,12 +71,27 @@ define('forum/topic', [
 		handleThumbs();
 
 		$(window).on('scroll', utils.debounce(updateTopicTitle, 250));
+		configurePostToggle();
 
 		handleTopicSearch();
 
 		hooks.fire('action:topic.loaded', ajaxify.data);
 	};
 
+	function configurePostToggle() {
```

Comment:
How hard would it be to fix the lint errors? Even though our whole GitHub actions might not pass, it might be helpful to have as many files as possible follow ESLint guidelines. If it's too much work for the checkpoint, we can work on fixing it by Friday